src/components/molecules/subheaderLinks/SubheaderLinks.tsx

```diff
@@ -0,0 +1,30 @@
+import { ListSubheader, ListItemButton, ListItemText } from '@mui/material';
+import { NavStyle } from '../../../common/utils/newStyle';
+
+interface ListButtonProps {
+  label: string;
+  filter: string;
+}
+
+interface SubheaderLinksProps {
+  title: string;
+  links: ListButtonProps[];
+  navigate: (route: string) => void;
+}
+
+export function SubheaderLinks({ title, links, navigate }: SubheaderLinksProps) {
+  return (
+    <ListSubheader component="div" sx={{ ...NavStyle }}>
+      {title}
+      {links.map(({ label, filter }) => (
+        <ListItemButton
+          key={filter}
+          onClick={() => navigate(`/videos?filter=${filter}`)}
```

Comment:
🔧 I don't think this will work fully. The `series` and `animation` genres don't go to this format of URL. They have their own paths (`/series` and `/series/low-phe-life` respectively).  Maybe pass the navigation destination in as a prop?